src/style.css

```diff
@@ -0,0 +1,52 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars .stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  background: url(./images/star.svg) no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star,
+.stars--2 .stars__star,
+.stars--3 .stars__star,
+.stars--4 .stars__star,
+.stars--5 .stars__star {
+  background: url(./images/star-active.svg) no-repeat;
+  background-position: center;
+}
```

Comment:
remove these it's useless, each star should have a simple star as default background, then use nth-child to add active star